routes/web.php

```diff
@@ -27,29 +27,9 @@
 /**
  * Add A New Task
  */
-Route::post('/task', function (Request $request) {
-    $validator = Validator::make($request->all(), [
-        'name' => 'required|max:255',
-    ]);
-
-    if ($validator->fails()) {
-        return redirect('/')
-            ->withInput()
-            ->withErrors($validator);
-    }
-
-    $task = new Task;
-    $task->name = $request->name;
-    $task->save();
-
-    return redirect('/');
-});
+Route::post('/task', [App\Http\Controllers\TaskController::class, 'postData'])->name('post.Data');
+Route::delete('/task/{id}', [App\Http\Controllers\TaskController::class, 'deleteTask'])->name('delete.Task');
```

Comment:
`App\Http\Controllers\TaskController` why use like this 
it should be 
`use App\Http\Controllers\TaskController;`